Terminal.Gui/Views/Label.cs

```diff
@@ -40,26 +48,27 @@ private void Label_TitleChanged (object sender, EventArgs<string> e)
         TextFormatter.HotKeySpecifier = HotKeySpecifier;
     }
 
-    /// <inheritdoc />
+    /// <inheritdoc/>
     public override string Text
     {
-        get => base.Title;
-        set => base.Text = base.Title = value;
+        get => Title;
+        set => base.Text = Title = value;
```

Comment:
Not sure I understand why Label has Title and Text as same but Button doesn't? But I guess that this pr doesn't change that just clarifies in comments.

Reply:
Consider this:

Right now `Border.Title` (which syncs `Border.Parent.Title`) is rendered in `Border.Thickness.Top`.

Once `Border` supports showing the Title as a subview, there's no reason it can't show up in `Border.Thickness.Left`.

Once that's done, why have `Label` at all?

Reply:
Not sure I understand but sounds like this is a stepping stone which is fine.

> Once that's done, why have Label at all?

Label is a very understandable thing for an end user so I think we should always have a class called that.  

But if it is a 1 line class that just configures its title to be subview then I'd be fine with that too.